### ISSUE TITLE
gnum4 is broken on staging

### ISSUE THREAD

**Issue by Mindavi** [NixOS/nixpkgs#136994]
### Describe the bug

The build of gnum4 is broken on staging. Probably broken in 9af75b49ecb672ee64a69ada43360dab89d08e59.

### Steps To Reproduce

`nix-build -A gnum4` on staging (c64a09f1eeea5a57849e74ca718a4dc9df1bc167)

### Expected behavior

Build succeeds.

### Additional context

<details><summary>Bisect log</summary>

```
git bisect start
# bad: [c64a09f1eeea5a57849e74ca718a4dc9df1bc167] Merge staging-next into staging
git bisect bad c64a09f1eeea5a57849e74ca718a4dc9df1bc167
# good: [da036efe0dd01d924d5c1532d4cec941b3fa023b] Merge pull request #135871 from abdelq/kmod-29
git bisect good da036efe0dd01d924d5c1532d4cec941b3fa023b
# good: [78503c4085759871735e77609eebe9aa9d3ef6bc] python3Packages.commoncode: 21.7.23 -> 21.8.27
git bisect good 78503c4085759871735e77609eebe9aa9d3ef6bc
# good: [2f5ea43e02a3ce3844fc28be2216cf92e26ae103] Merge master into staging-next
git bisect good 2f5ea43e02a3ce3844fc28be2216cf92e26ae103
# good: [2be182ab81d7cac34e01ad8858658b571bbb7e16] Merge pull request #136829 from r-ryantm/auto-update/bibletime
git bisect good 2be182ab81d7cac34e01ad8858658b571bbb7e16
# good: [42155910a0a499e9cc937ae89f022fcd5b551cf6] pkgsStatic.curl: fix build
git bisect good 42155910a0a499e9cc937ae89f022fcd5b551cf6
# good: [00f361a84625c1db9b3f4ed91ee9f35465133474] Merge pull request #129074 from bobby285271/pr11
git bisect good 00f361a84625c1db9b3f4ed91ee9f35465133474
# good: [5c0a3345a8001e9112404332920fad6caeea19e3] Merge pull request #136720 from alerque/vcsh
git bisect good 5c0a3345a8001e9112404332920fad6caeea19e3
# good: [4755c306472dbecb9422332fb5d77a3000e26594] Merge pull request #136915 from ncfavier/manualPages
git bisect good 4755c306472dbecb9422332fb5d77a3000e26594
# bad: [9af75b49ecb672ee64a69ada43360dab89d08e59] Merge remote-tracking branch 'upstream/staging-next' into staging
git bisect bad 9af75b49ecb672ee64a69ada43360dab89d08e59
# good: [745cf36f95eab6f8a50bb5d41470000f431f51d7] Merge branch 'staging-next-fix' into staging-next
git bisect good 745cf36f95eab6f8a50bb5d41470000f431f51d7
# good: [f238a66a7cd9a875ce72c9e2f56a5ed2abfe865f] Merge #136970: nixos/doc: resolve conflicts
git bisect good f238a66a7cd9a875ce72c9e2f56a5ed2abfe865f
# first bad commit: [9af75b49ecb672ee64a69ada43360dab89d08e59] Merge remote-tracking branch 'upstream/staging-next' into staging
```

</details>

### Notify maintainers

cc @lukegb which did the merge that seems to be the culprit.

### Metadata

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.13.12, NixOS, 21.11 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20210802_47e96bb`
 - channels(root): `"nixos-21.11pre312125.21c937f8cb1, nixpkgs-21.11pre303315.16105403bdd"`
 - channels(rick): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Maintainer information:
```yaml
attribute: gnum4
```

**Comment by lukegb** [comment]
Yeah, this merge seems broken in a number of respects; it rolls a bunch of things back because it merged in 9af75b49ecb672ee64a69ada43360dab89d08e59...

**Comment by lukegb** [comment]
OK, this should build again: I get /nix/store/f3y52hqby0m4pxx4fwp3kia32lc0h8lm-gnum4-1.4.19, which is what we were at at da036efe0dd01d924d5c1532d4cec941b3fa023b.
